llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -9010,6 +8994,20 @@ bool VPRecipeBuilder::getScaledReductions(
   return false;
 }
 
+VPIRMetadata VPRecipeBuilder::getRecipeMetadata(Instruction *I) const {
```

Comment:
Should this simply `return VPIRMetadata(I, LVer);`, letting VPIRMetadata take care of how to construct itself given both `I` and `LVer`?

Reply:
Moved, thanks!